src/metabase/models/metric.clj

```diff
@@ -65,7 +65,7 @@
 
 ;; ## Persistence Functions
 
-(defn create-metric
+(defn create-metric!
```

Comment:
I'll let you decide what you want to do here so that you can merge this PR because it doesn't really matter that much to me, but I still don't see the relevance of this notation.  Every function in Metabase or in the general clojure ecosystem that modifies something in a database would be suffixed with an exclamation mark?